DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/credit-and-debit-cards/mastercard/how-does-mastercard-work.md

read_more: '.'
---
## How it works

> >Mastercard payments are processed as a credit card payment. Therefore, credit card payments must be enabled within your MultiSafepay Control to process a Mastercard payment. Submit your request to enable credit cards (Visa, Mastercard, American Express and Maestro) to your account manager or email us at <sales@multisafepay.com>

Reviewer: ```suggestion
>Mastercard payments are processed as credit card payments. To enable credit card payments in your MultiSafepay Control, email your account manager at <sales@multisafepay.com>
```

* If you refund the transaction (partially) on the same day this is called a reversal. To keep things easy manageable in your reporting, this is logged as a refund. However, your customers might experience otherwise: the transaction in their overview will be adjusted to the new amount (partial reversal) of the order __or__ is not debited at all (full reversal).

>     If a different currency is desired, contact our support department for the possibilities. Email your request to <support@multisafepay.com>

Reviewer: ```suggestion
    To support another currency, email the Support Team at <support@multisafepay.com>
```
